client/my-sites/posts/post.jsx

```diff
@@ -381,4 +380,4 @@ export default connect(
 		};
 	},
 	{ setPreviewUrl, setLayoutFocus }
-)( localize( Post ) );
+)( updatePostStatus( localize( Post ) ) );
```

Comment:
I'll still be touching `my-sites/posts` for a little while, so I'll for sure tackle this later 💯